##HerkuleX_Info Array_RAM

int8 index
MsgHerkulexRam[] pherkulex_ram

